apps/explorer/src/components/common/TableSearch/styled.ts

```diff
@@ -0,0 +1,50 @@
+import { Media, Color } from '@cowprotocol/ui'
+
+import { Wrapper, Input, SearchIcon } from 'explorer/components/common/Search/Search.styled'
```

Comment:
Wrong import

Reply:
Imports might be relative inside the project

Reply:
should be `components/common/....` I take?